pages/gallery.html

```diff
@@ -14,16 +14,58 @@
     <title>Galeria - Hostal las paisas</title>
   </head>
   <body>
-    <nav>
-      <!-- nav bar section -->
-    </nav>
     <section class="small-hero small-hero__image--gallery ">
       <div class="small-hero__layer"></div>
       <div class="small-hero__layer small-hero__layer--darker"></div>
       <h1 class="small-hero__title">Galería</h1>
     </section>
+    <nav>
+      <!-- nav bar section -->
+    </nav>
+    <!-- gallery -->
+    <section class="container container-componet gallery">
```

Comment:
what is `container-componet`? what does this do?

Reply:
gives the padding to the container

Reply:
maybe the gallery class could give this property.